client/fuzz_test.go

```diff
@@ -28,34 +26,31 @@ func (fz *fuzzSuite) FuzzQueryBalance(f *testing.F) {
 	fz.Require().Equal("hello", testRes.Message)
 
 	// 1. Generate some seeds.
-	bz, err := fz.cdc.Marshal(&types.QueryBalanceRequest{
```

Comment:
Sadly this won't give the fuzzer good direction and it'll take a long while to figure out how a request will look like. Perhaps this is temporary?

Reply:
we can move the fuzzing into the bank module but cant keep it at the root level